content/blog/2020-01-vroom-1-1-0/index.Rmarkdown

```diff
@@ -0,0 +1,131 @@
+---
+title: vroom 1.1.0
+author: Jim Hester
+date: '2020-01-14'
+slug: vroom-1-0-0
+description: vroom 1.1.0 is now on CRAN!
+categories:
+  - package
+tags: [package, r-lib]
+photo:
+  url: https://www.pexels.com/photo/12801/
+  author: Chris Peeters
+---
+
+```{r include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+I'm excited to announce that [vroom 1.1.0](http://vroom.r-lib.org) is now
+available on CRAN!
+
+vroom reads rectangular data, such as comma separated (csv), tab separated (tsv) or fixed width files (fwf) into R.
+It performs similar roles to functions like [`readr::read_csv()`](http://readr.r-lib.org), [`data.table::fread()`](http://r-datatable.com) or `read.csv()`.
+But for many datasets `vroom::vroom()` can read them much, much faster (hence the name).
+Get the latest version with:
+
+```{r, eval = FALSE}
+install.packages("vroom")
+```
+
+And attach the package by running
+
+```{r}
+library(vroom)
+```
+
+Improvements in this release include a hex logo, support for big integer data, improved delimiter guessing, including delimiters in specifications, and streamlined reading from standard input.
+
+See the [change log](https://vroom.r-lib.org/news/index.html) for a full list of changes and bug fixes in this version.
+
+## Hex logo
+
+Thanks to [Allison Horst](https://github.com/allisonhorst/stats-illustrations) we now have an awesome hex logo for vroom!
+
+![](logo.png)
+
+## Big integer support
+
+R's standard integers are stored in 32 bits of binary data, which means that the largest value they can store is 2,147,483,647 (`2^31 - 1`).
+R implicitly converts integers for most operations with doubles to 64 bit floating point values, which is why you may not have noticed this limitation before.
```

Comment:
```suggestion
R implicitly converts integers for most operations with doubles to 64-bit floating point values, which is why you may not have noticed this limitation before.
```

Reply:
_I_ would hyphenate that grammatically, but if it's simply not what computer-y people do (à la the non-hyphenated ‘object oriented programming’), then ignore.